thijsvanloef/palworld-server-docker
# Added description for server

## Context 

* Add a description to the server

## Choices

* Implemented the same as the other settings

## Test instructions

1. Set a description in docker-compose
2. Start container 
3. Join game
4. Go to the options in game to see the description. For my testing I did `Hello World`.
![20240125214337_1](https://github.com/thijsvanloef/palworld-server-docker/assets/18587674/29f12645-27f8-4232-bd20-8248859b5009)



## Checklist before requesting a review
